.github/workflows/scan_for_secrets.yaml

```diff
@@ -18,3 +18,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITLEAKS_LICENSE: ${{ secrets.GITLEAKS_LICENSE}}
+        with:
+          baseline-path: .github/workflows/gitleaks-baseline.json
```

Comment:
Can you move this up one directory into the `.github`? It's become somewhat the catch all for workflow related files and it makes it easier to find not being in with all the workflow files.

Reply:
Is there a way to whitelist specific parts of files? The config.ts file with the test jwt for example would be nice to ignore. The dev postgres password "example". Etc.

Reply:
Moved the file as suggested.

You can exclude parts of a file via [`gitleaks:allow`](https://github.com/gitleaks/gitleaks?tab=readme-ov-file#gitleaksallow) comments.